src/routes/Onboarding/index.tsx

```diff
@@ -112,25 +112,24 @@ export function useDemoCode() {
 
 export function useNextClick(newStatus: string) {
   const filePath = useAbsoluteFilePath()
-  const {
-    settings: { send },
-  } = useSettingsAuthContext()
   const navigate = useNavigate()
 
   return useCallback(() => {
-    send({
+    settingsActor.send({
       type: 'set.app.onboardingStatus',
       data: { level: 'user', value: newStatus },
     })
     navigate(filePath + PATHS.ONBOARDING.INDEX.slice(0, -1) + newStatus)
-  }, [filePath, newStatus, send, navigate])
+  }, [filePath, newStatus, settingsActor.send, navigate])
 }
 
 export function useDismiss() {
   const filePath = useAbsoluteFilePath()
-  const {
-    settings: { state, send },
-  } = useSettingsAuthContext()
+  const settings = useSettings()
+  const send = settingsActor.send
```

Comment:
Nit: In the future I advocate leaving calls as the full `settingsActor.send`, it's way easier to reason about / qualify

Reply:
I agree, this was just a diff reduction approach.